frontend/src/components/Forms/TeamEventCreditsForm/TeamEventsCreditDashboard.tsx

```diff
@@ -28,6 +35,34 @@ const TeamEventCreditDashboard = (props: {
     return 1;
   };
 
+  const deleteTECAttendanceRequest = (attendance: TeamEventAttendance) => {
+    // const deleteTECAttendanceRequest = () => {
+
+    TeamEventsAPI.deleteTeamEventAttendance(attendance.uuid)
+      .then(() => {
+        setShowComponent(false);
+
+        Emitters.generalSuccess.emit({
+          headerMsg: 'Team Event Attendance Deleted!',
+          contentMsg: 'Your team event attendance was successfully deleted!'
+        });
+        ImagesAPI.deleteEventProofImage(attendance.image);
+        Emitters.teamEventsUpdated.emit();
+      })
+      .catch((error) => {
+        Emitters.generalError.emit({
+          headerMsg: 'You are not allowed to delete this team event attendance!',
+          contentMsg: error
+        });
+      });
```

Comment:
Try something like

`setPendingAttendance(pendingAttendance.filter((currAttendance) => currAttendance.uuid !== attendance.uuid))` right here in this `deleteTECAttendanceRequest` function, right where `setShowComponent(false);` is (and delete `setShowComponent` component, since we won't need it.

You will need to pass in `setPendingAttendance` as a prop into this component.

i.e.

```
const TeamEventCreditDashboard = (props: {
  allTEC: TeamEventInfo[];
  approvedAttendance: TeamEventAttendance[];
  pendingAttendance: TeamEventAttendance[];
  isAttendanceLoading: boolean;
  // add the following line
  setPendingAttendance: Dispatch<SetStateAction<TeamEventAttendance[]>>;
```

and then

```
<TeamEventCreditDashboard
          allTEC={teamEventInfoList}
          approvedAttendance={approvedAttendance}
          pendingAttendance={pendingAttendance}
          isAttendanceLoading={isAttendanceLoading}
          // add the following line
          setPendingAttendance={setPendingAttendance}
        />
```